instance/static/html/instance/index.html

```diff
@@ -0,0 +1,77 @@
+<top-bar>
+  <ul class="title-area">
+    <li class="name">
+      <h1><a href="#">OpenCraft - Instances Manager</a></h1>
```

Comment:
@bradenmacdonald The way I think of it is that "OpenCraft" is a suite of tools, including the instance manager, which allows to automate the things we do. So technically that's already the case - the name for the current code would be "OpenCraft Instance Manager". I'll edit the title to remove the dash, and the "s" : ) And +1 for including the logo, yup!

As a side note, this gives me an occasion to explain the larger goal with this code -- to start crystallizing our workflow and ways of working into code. We're pretty unique and good at that, in many ways (decentralized, working openly on free software, focus on quality and contributions, good development & deployment workflow inspired from edX good practices...). 

Other companies & communities could benefit from it. In that context, the reason to call the suite of tools to which the IM belongs "OpenCraft", like the company, is that the two aren't necessarily distinct. Ie it's a way to try to scale some of what we are doing manually, and become more than a development shop - ie focus our brand/name on the product, rather than on the company (like GitHub is called GitHub, not GitHub Repository Manager : ) ). Does that make sense, or am I just full of $%$? : ) Note that's more a long term dream plan than a current reality - for now I'm fine with using OpenCraft IM, since we are currently, as a matter of fact, a development shop. We just don't have to be that forever.